src/Radicle/Internal/Pretty.hs

```diff
@@ -50,7 +50,8 @@ instance (Copointed t, Ann.Annotation t) => Pretty (Ann.Annotated t ValueF) wher
                         , sep $ pretty <$> toList vals
                         ])
       where
-        escapeStr = T.replace "\"" "\\\"" . T.replace "\\" "\\\\"
+        -- We print string literals escaped just like Haskell does.
```

Comment:
```suggestion
        -- Radicle prints string literals escaped just like Haskell does.
```

Reply:
I suggested to myself to change this comment.

Reply:
```suggestion
        -- We print string literals escaped just like Haskell does.
```

Reply:
No idea what that just did, seems you can just add another suggestion instead. But it can't be applied until the first one is?

Reply:
Ah it was invalid because no changes.
```suggestion
        -- Everyone prints string literals escaped just like Haskell does.
```